### PR TITLE
[FIX] point_of_sale: fix strange mail construct

### PR DESCRIPTION
**Pull request by tde-banana-odoo**

### CODE REVIEW

**Comment by tde-banana-odoo** [comment]
@pimodoo 

cc @mart-e

File: addons/point_of_sale/models/pos_order.py [hunk 1]
```diff
@@ -616,17 +616,24 @@ def refund(self):
 
     @api.model
     def action_receipt_to_customer(self, name, client, ticket, order_ids=False):
-        template_obj = self.env['mail.mail']
-        message = "<p>Dear %s,<br/>Here is your electronic ticket from the %s. </p>" % (client['name'], name)
+        orders = self.browse(order_ids) if order_ids else self
+        if not orders:
```
Comment (pimodoo) [inline addons/point_of_sale/models/pos_order.py]: So I've just checked, and order_ids seems to be always sent to the function (before it was only when invoiced, but now it's always). So I can change it in multi for master, and maybe just check the group here.

I'll change it in master once this one is forwardported.

**Comment by tde-banana-odoo** [comment]
Code updated, it checks for the group_pos_user group and then just browse given ids or use a void record set in order to avoid side effects in stable. In master indeed improving it as an ensure one multi will be even better.

**Comment by pimodoo** [comment]
@tde-banana-odoo looks good to me, I'll do the improvement in master if you want.

**Comment by tde-banana-odoo** [comment]
@robodoo r+

@pimodoo well I can do them when forward bot proposes the Fwd PR, I will ping you :) .

**Pull request closed without merging**